src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -273,6 +273,9 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
         this._eventManager.listen(this._videoElement, Html5EventType.ENDED, () => this._clearHeartbeatTimeout());
         this._eventManager.listen(this._videoElement, Html5EventType.ABORT, () => this._clearHeartbeatTimeout());
         this._eventManager.listen(this._videoElement, Html5EventType.SEEKED, () => this._onSeeked());
+        // Sometimes when playing live in safari and switching between tabs the currentTime goes back with no seek events
+        this._eventManager.listen(window, 'focus', () => setTimeout(() => this._onSeeked(), 1000));
```

Comment:
need to think if all of this check is relevant for live, so check if we even detect stalling/waiting when doing throttling test.
If we don't then just cancel the onTimeUpdate check when we are in live.

Reply:
Yes. `onTimeUpdate` check is relevant for live as well. 
We have to detect internet disconnection for live to get an error like in chrome